gno.land/cmd/gnoland/testdata/getting_started.txtar

```diff
@@ -0,0 +1,17 @@
+# getting started
+
+## start gnoland chain
+gnoland-start
+
+## should show test1
+gnokey list
```

Comment:
this one is unecessary; keep it if you want, but feel free to break it if it allows making the other gnokey commands more straightforward.

Reply:
sure, I just use this to confirm that the test1 account was correctly setup, I will remove it